dataservice/api/family_relationship/models.py

```diff
@@ -57,6 +58,8 @@ class FamilyRelationship(db.Model, Base):
                                                       nullable=False)
 
     participant2_to_participant1_relation = db.Column(db.Text())
+    source_text_notes = db.Column(db.Text(),
```

Comment:
What kind of notes are these exactly?

Reply:
I haven't seen it very often, but I think they're supposed to be anything the investigator included in their pedigree file describing the relationship.

Reply:
ya, for example, Engle data has special notes for some relations.